Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -2645,7 +2641,7 @@ internal static void Update(InputUpdateType updateType)
             }
         }
 
-#if UNITY_EDITOR && ENABLE_XR_COMBINED_DEFINE
+#if UNITY_EDITOR
```

Comment:
@jimon
Pointing Dmytro this way. I landed this runUpdatesInEditMode property on the condition that it's something we use specifically for XR for now, and can expand it to other platforms if needed, and the feature is more fully tested for those platforms.

I think this should still have some kind of XR guard.

Reply:
the trick with this one is that we currently have no XR specific defines here other than ENABLE_VR which will be 1 in the editor anyway. there's no asmdef for the test project that i can see either. could add one? @jimon ?

Reply:
Yes true, this functionality should stay XR-only for near future, as running-in-editor was not properly tested on our side, we were thinking to give it a deeper pass and then enable as a proper feature.

@mattdatwork are XR modules always on in the editor? anything else we can use? I assume editor is not in XR mode by default.

Reply:
What about adding it if XR Management is included

Reply:
editor is ENABLE_VR by default. which is why I left it on. 
the trick here is that theres no current _test_ asmdef. so we'd need to add one to put in any compile defines for the tests.

Reply:
The condition here is **NOT** tests.
The condition is XR systems.  XR Management is the candidate I suggest.

Reply:
<facepalm> right. sorry thought this was the test file. ill swap that over to ENABLE_VR as well.